### PR TITLE
Feat/minh tf fizz buzz

### PR DESCRIPTION
**Pull request by hlminh2000**

### CODE REVIEW

File: _posts/minh_ha/2018-06-22-Tensorflow-fizz-buzz.md [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+<image src="{{ site.urlimg }}/minh_ha/ai_robot.jpg" />
+
+# Introduction
+
+If you have been remotely involved in the tech industry, you must have been
+hearing about the rise of AI and Machine Learning (ML) at every corner you turn to.
```
Comment (Fgerthoffert) [inline _posts/minh_ha/2018-06-22-Tensorflow-fizz-buzz.md]: you can probably remove "you turn to".

File: _posts/minh_ha/2018-06-22-Tensorflow-fizz-buzz.md [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+The mix of excitement in the potential of what it can do, together with the fear
+of the AI overlord, has been keeping many of us developers up at night.
+So when google announced the release of the long-waited [TensorFlow.js](https://js.tensorflow.org/)
+(a Javascript implementation of their ever popular machine learning library), I gave myself no
+choice but to dive right in and solve one of the software world's most infamous
+problem: **Fizz buzz!**
```
Comment (Fgerthoffert) [inline _posts/minh_ha/2018-06-22-Tensorflow-fizz-buzz.md]: Actually :), I didn't know what the fizz buzz was, could you link it to wikipedia: https://en.wikipedia.org/wiki/Fizz_buzz ?

File: _posts/minh_ha/2018-06-22-Tensorflow-fizz-buzz.md [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+   ```
+
+2) **Set up the model:**
+   I went ahead and set up an [artificial neural network](https://en.wikipedia.org/wiki/Artificial_neural_network),
+   the spooky "simulation of a human brain" that has been in the news. With Tensorflow,
+   the code may look something like so:
+
+   ```javascript
+     const fizzBuzzModel = tf.sequential();
+     fizzBuzzModel.add(tf.layers.dense({ units: 10, inputShape: 1 }));
```
Comment (Fgerthoffert) [inline _posts/minh_ha/2018-06-22-Tensorflow-fizz-buzz.md:131]: At that point, I wonder what "relu" and "softmax" are or mean. I see those are listed in the above wikipedia page, but maybe a very brief sentence, not necessarily on what both are, but on what is "activation".

**Review by Fgerthoffert — APPROVED** [review]
Great article !

File: _posts/minh_ha/2018-06-22-Tensorflow-fizz-buzz.md (hunk 3 above)
Comment (hlminh2000) [reply _posts/minh_ha/2018-06-22-Tensorflow-fizz-buzz.md:131]: Thanks for the feedback @Fgerthoffert :) I've added some clarification and links to resources

**Review by andricDu — APPROVED** [review]
Excellent!
